docs/how-to-guides/integrating-autoware-with-your-vehicle.md

```diff
@@ -170,4 +170,76 @@ Now the vehicle should drive along the calculated path!
 
 You may need to tune your parameters depending on the domain in which you will operate your vehicle.
 
+## 8. Customize your own package based on Autoware-msgs / Autoware-package
+
+In many practical applications, apart from the available nodes / modules of Autoware, you may have the need to create your own packages which communicate with Autoware nodes or utilize some Autoware implementations (such like math methods including A-star, interpolation, mpc algorithm and so on). In this case, you can follow the instructions below to customize your specific package.
+
+### Package using Autoware-msgs
+
+Since Autoware is built on ROS (Autoware.Universe / Autoware.Core on ROS2), if you have the urge to communicate with other Autoware nodes, then you are supposed to obey the rule of node subscribing / publishing messages via topic in specified message type. Please refer to the [ROS Tutorial](https://docs.ros.org/en/humble/Tutorials.html) for details.
```

Comment:
[style] Follow Google developer documentation style guide.
- https://developers.google.com/style/link-text?hl=en#structure-link-text
- https://developers.google.com/style/tone#politeness-and-use-of-please

```suggestion
Since Autoware is built on ROS (Autoware.Universe / Autoware.Core on ROS2), if you have the urge to communicate with other Autoware nodes, then you are supposed to obey the rule of node subscribing / publishing messages via topic in specified message type. For details, refer to the [ROS Tutorial](https://docs.ros.org/en/humble/Tutorials.html).
```